.github/workflows/static-checks.yml

```diff
@@ -31,22 +31,20 @@ jobs:
                   path: ~/.npm
                   key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
 
-            - name: Npm install and build
+            - name: Npm install
               # A "full" install is executed, since `npm ci` does not always exit
               # with an error status code if the lock file is inaccurate.
               #
               # See: https://github.com/WordPress/gutenberg/issues/16157
-              run: |
-                  npm install
-                  npm run build
+              run: npm install
 
             - name: Lint JavaScript and Styles
               run: npm run lint
 
-            - name: Type checking
-              run: npm run build:package-types
+            - name: Build and type checking
+              run: npm run build
```

Comment:
We probably don't even need to run the full build in this workflow? Just `build:package-types` is enough?

Reply:
I removed the build step. We can add it back if we really need it.